PubnativeLite/PubnativeLite/Interstitial/HyBidInterstitialAd.m

```diff
@@ -87,6 +87,13 @@ - (void)load {
     }
 }
 
+- (void)setSkipOffset:(NSInteger)seconds
```

Comment:
Based on my comment at HyBidinterstitialAd.h, you wont need this method.

Reply:
@can-soykarafakili I'll keep the method, as I'm doing some validations there and make the property private.